CHANGELOG.md

```diff
@@ -4,7 +4,9 @@
 
 ## 💡 Enhancements 💡
 
-- `tailsampling` processor: Add new policy `latency` (#3750)
+- `tailsampling` processor:
+  - Add new policy `latency` (#3750)
```

Comment:
I guess the latency made it to 0.29.0, no?

Reply:
Actually... 0.29.0 hasn't been released for contrib yet. I heard yesterday that the release train has departed already, though.

Reply:
Is it still possible to get this PR in 0.29 or is the release already cut?

Reply:
I'm not sure how to avoid a merge conflict with the release PR, should I wait until #3863 is merged to rebase and update the changelog?

Reply:
I think @bogdandrutu mentioned yesterday that they were having build problems, blocking the release. I don't expect any new features to be included for 0.29.0. Wait for the release, then rebase and update the changelog.